src/controller/storage-provider-report/storage-provider-report.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Controller, Get, Logger } from '@nestjs/common';
+import { ApiOkResponse, ApiOperation } from '@nestjs/swagger';
+import { AggregatedProvidersIPNIReportingStatus } from '../../service/ipni-misreporting-checker/types.ipni-misreporting-checker';
+import { IpniMisreportingCheckerService } from '../../service/ipni-misreporting-checker/ipni-misreporting-checker.service';
+
+@Controller('providerReport')
+export class StorageProviderReportController {
+  private readonly logger = new Logger(StorageProviderReportController.name);
+
+  constructor(
+    private readonly ipniMisreportingCheckerService: IpniMisreportingCheckerService,
+  ) {}
+
+  @Get('/aggregatedIpniStatus')
```

Comment:
```suggestion
  @Get('/aggregated-ipni-status')
```
For consistency with other endpoints.

Also, I think it fits better in `/stats/providers` than in `/providerReport`

Reply:
Oh, actually I see that while most endpoints use `-`, we have `clientReport` and `allocatorReport` :/
Let's decide on one convention and stick to it. I vote for `-`. Uppercase in URLs is controversial topic :D

Reply:
100% agree. Not sure what to do with this, because changing `clientReport` etc will be breaking change

Reply:
Let's say I'll use `-` for all new endpoints even if it breaks consistency withing single endpoint URL

Reply:
> 100% agree. Not sure what to do with this, because changing clientReport etc will be breaking change

Can we bind these controllers to multiple routes? So that both `clientReport` and `client-report` work?
We could do that for a transition period, ask the UI to update and drop the old one when it's no longer used by anyone.

Reply:
Ok, let's do transition period, I'll use redirects for this